examples/chip-tool/README.md

```diff
@@ -120,6 +120,45 @@ The endpoint id must be between 1 and 240.
 
 The client will send a single command packet and then exit.
 
+## Configuring the client for Group Commands
+
+Prior to sending a Group command, both the end device and the Client (Chip-tool)
+must be configured appropriately.
+
+To configure the client please use the groupsettings option
+
+    $ chip-tool groupsettings
+
+A group with a valid encryption key needs to be set. The groupid and the
+encryption key must match the one configured on the end device.
+
+To add a group
+
+    $ chip-tool groupsettings add-group TestName 0x1010
+
+To add a keyset
+
+    $ chip-tool groupsettings add-keyset 0xAAAA 0 000000000021dfe0d0d1d2d3d4d5d6d7d8d9dadbdcdddedf
+
+Take note that the epoch key must be in hex form without the '0x' prefix
```

Comment:
This does not document that the first 16 chars will be treated as the validity time.

But why not just pass the validity time as a separate arg to start with instead of sticking it in the hex buffer?

Reply:
fixed

Reply:
fixed